week2/homework/src/add.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/** ---------------------------------------
+     Base
+ ---------------------------------------- */
+const fs = require('fs');
+
+/** ---------------------------------------
+     Functions for Command
+ ---------------------------------------- */
+function fillEmptyDatabase() {
+  const content = fs.readFileSync('To-Do Database.json', 'utf8');
+  switch (content) {
```

Comment:
Switch is used when you have multiple scenario's (condition). If there is only 1 condition it makes more sense to use an `if` statement

Reply:
It's good to learn that. All switches have been switched to if statement.